### PR TITLE
Parameters and operators doc revamp

### PR DESCRIPTION
**Pull request by candida-d**

### CODE REVIEW

File: docs/node-operators/get-started/setup-non-validator.md [hunk 1]
```diff
@@ -103,30 +94,29 @@ This creates a set of configuration files which you then need to alter for your
...
 8. Open the config file `$TENDERMINT_PATH/config/config.toml` and update the value `[p2p] -> persistent_peers` with the list created above
 
 9. Set the following values:
 
 ```toml
   [p2p]
-    max_packet_msg_payload_size=16384
     pex = true
-    seeds = "b0db58f5651c85385f588bd5238b42bedbe57073@13.125.55.240:26656,abe207dae9367995526812d42207aeab73fd6418@18.158.4.175:26656,198ecd046ebb9da0fc5a3270ee9a1aeef57a76ff@144.76.105.240:26656,211e435c2162aedb6d687409d5d7f67399d198a9@65.21.60.252:26656,c5b11e1d819115c4f3974d14f76269e802f3417b@34.88.191.54:26656,61051c21f083ee30c835a34a0c17c5d1ceef3c62@51.178.75.45:26656,b0db58f5651c85385f588bd5238b42bedbe57073@18.192.52.234:26656,36a2ca7bb6a50427be2181c8ebb7f62ac62ebaf5@m2.vega.community:26656,9903c02a0ff881dc369fc7daccb22c1f9680d2dd@api0.vega.community:26656,9903c02a0ff881dc369fc7daccb22c1f9680d2dd@api0.vega.community:26656,32d7380b195c088c0605c5d24bcf15ff1dade05f@api1.vega.community:26656,4f26ec99d3cf6f0e9e973c0a5f3da87d89ec6677@api2.vega.community:26656,eafacd11af53cd9fb2a14eada53485779cbee4ab@api3.vega.community:26656"
+    seeds = "insert-network-seeds-here"
```
Comment (daniel1302) [inline docs/node-operators/get-started/setup-non-validator.md:106]:
```suggestion
    persistent_peers = "insert-network-peers-here"
```

Let's change it to the persistent peers, as according to the documentation persistent peers also participate in peers exchange when the `pex` is set to true.

File: docs/node-operators/get-started/setup-validator.md [hunk 2]
```diff
@@ -143,9 +142,9 @@ Use a service manager (for example Systemd) to run Visor. While you can use any
 ```
 
 ## Modify Vega config
-When announcing your node (below), the node will need to be pointing to a Tendermint node. Set that in your Vega config based on information in the Tendermint config file. 
+When announcing your node (below), the node will need to be pointing to a CometBFT node. Set that in your Vega config based on information in the CometBFT config file. 
 
-Find the address for the RPC server in your Tendermint config `YOUR_TENDERMINT_HOME_PATH/config/config.toml`, under the heading `TCP or UNIX socket address for the RPC server to listen on`. 
+Find the address for the RPC server in your CometBFT config `YOUR_COMETBFT_HOME_PATH/config/config.toml`, under the heading `TCP or UNIX socket address for the RPC server to listen on`.
```
Comment (daniel1302) [inline docs/node-operators/get-started/setup-validator.md:147]: We have to add also something like the following in the `Point to Arbitrum node` (see the issue I have created here: https://github.com/vegaprotocol/vega/issues/11684) :


Add config for the Event Forwarded for the Arbitrum endpoint:

```toml title=YOUR_VEGA_HOME/config/node/config.toml
...
[EvtForward]
...


  [[EvtForward.EVMBridges]]
    Level = "Info"
    MaxEthereumBlocks = 500
    PollEventRetryDuration = "500ms"
    ChainID = "421614"
    SkipClientVerification = true
    HeartbeatIntervalForTestOnlyDoNotChange = "0s"
```

File: docs/node-operators/get-started/setup-validator.md [hunk 3]
```diff
@@ -345,32 +342,30 @@ vega nodewallet import --chain=ethereum --home="YOUR_VEGA_HOME_PATH" --wallet-pa
...
 
-Update the tendermint config located at `YOUR_TENDERMINT_HOME_PATH/config/config.toml` and set the `persistent_peers` field under the `[p2p]` section.
+Update the tendermint config located at `YOUR_COMETBFT_HOME_PATH/config/config.toml` and set the `persistent_peers` field under the `[p2p]` section.
 
 Here's an example:
 
 ```toml
 [p2p]
-persistent_peers = "55b8ac477ddd6c0c9bae411dfa6ebfb46e7b4022@veganodeoperator.com:26656,2d1bbf1229bd7f8e57e89c61346ab6928d61881b@127.0.0.1:26656"
+persistent_peers = "55b8ac477ddd6c0c9bae411dfa6ebfb46e7b4022@nodeoperator.com:26656,2d1bbf1229bd7f8e57e89c61346ab6928d61881b@127.0.0.1:26656"
```
Comment (daniel1302) [inline docs/node-operators/get-started/setup-validator.md:357]: Below replace 

```
Under `Mempool Configuration Option`, ensure that `broadcast = true`.
```

to

```
Under `mempool`, ensure that `broadcast = true`.
```

**Review by daniel1302 — COMMENTED** [review]
I have left a few comments,rest looks good, I have also tested the procedure and works well.

File: versioned_docs/version-v0.77/node-operators/get-started/setup-non-validator.md [hunk 4]
```diff
@@ -103,30 +94,29 @@ This creates a set of configuration files which you then need to alter for your
...
 8. Open the config file `$TENDERMINT_PATH/config/config.toml` and update the value `[p2p] -> persistent_peers` with the list created above
 
 9. Set the following values:
 
 ```toml
   [p2p]
-    max_packet_msg_payload_size=16384
     pex = true
-    seeds = "b0db58f5651c85385f588bd5238b42bedbe57073@13.125.55.240:26656,abe207dae9367995526812d42207aeab73fd6418@18.158.4.175:26656,198ecd046ebb9da0fc5a3270ee9a1aeef57a76ff@144.76.105.240:26656,211e435c2162aedb6d687409d5d7f67399d198a9@65.21.60.252:26656,c5b11e1d819115c4f3974d14f76269e802f3417b@34.88.191.54:26656,61051c21f083ee30c835a34a0c17c5d1ceef3c62@51.178.75.45:26656,b0db58f5651c85385f588bd5238b42bedbe57073@18.192.52.234:26656,36a2ca7bb6a50427be2181c8ebb7f62ac62ebaf5@m2.vega.community:26656,9903c02a0ff881dc369fc7daccb22c1f9680d2dd@api0.vega.community:26656,9903c02a0ff881dc369fc7daccb22c1f9680d2dd@api0.vega.community:26656,32d7380b195c088c0605c5d24bcf15ff1dade05f@api1.vega.community:26656,4f26ec99d3cf6f0e9e973c0a5f3da87d89ec6677@api2.vega.community:26656,eafacd11af53cd9fb2a14eada53485779cbee4ab@api3.vega.community:26656"
+    seeds = "insert-network-seeds-here"
```
Comment (candida-d) [inline versioned_docs/version-v0.77/node-operators/get-started/setup-non-validator.md:106]:
```suggestion
    persistent_peers = "insert-network-peers-here"
```

**Review by daniel1302 — APPROVED** [review]

**Pull request merged**